tests/testthat/test-textstat_proxy.R

```diff
@@ -254,8 +254,8 @@ test_that("textstat_proxy works with non-intersecting documents or features", {
 
 test_that("raises error when dfm is empty (#1419)", {
     mt <- dfm_trim(data_dfm_lbgexample, 1000)
-    expect_error(textstat_proxy(mt),
-                 quanteda:::message_error("dfm_empty"))
+    expect_silent(textstat_proxy(mt))
```

Comment:
I like the error message better, otherwise you are saying that all empty features have a distance or similarity of zero, when in fact you don't know a thing about them.

Reply:
If you like it, we should change it to warn instead of error, because error breaks programs that depends on the function in extreme cases.

Reply:
As long as we return NA for the distance or similarity matrix when the documents or features are empty, I'm fine with NA and no warning for `textstat_simil()` and `textstat_dist()`.